### ISSUE TITLE
[BUG] Doppler setup isn't working.

### ISSUE THREAD

**Issue by anirudhsama** [DopplerHQ/cli#385]
**Describe the bug**
Doppler setup doesn't start interactive flow.

**To Reproduce**
Steps to reproduce the behavior. Please include output from running the command with `--debug`.
```
doppler setup --debug
Debug: Using config dir /Users/ani/.doppler
Debug: Using config file /Users/ani/.doppler/.doppler.yaml
Debug: Reading config file
Debug: Retrieving token from system keyring
Debug: Sending anonymous analytics payload: '{"command":"doppler.setup"}'
Debug: Performing HTTP POST to https://cli.doppler.com/v1/analytics
Debug: Retrieving token from system keyring
```

**Expected behavior**
`doppler config` should start an interactive wizard asking to select the project and configuration.

**Screenshots**
<img width="614" alt="CleanShot 2023-04-21 at 16 53 15@2x" src="https://user-images.githubusercontent.com/832760/233624148-588440e8-e1a9-45c2-a2f1-cef07d25e3ff.png">

**Desktop (please complete the following information):**
 - OS: macOS 
 - Version 13.3

**CLI Version:**
Version 3.57.0

**Additional context**

**Comment by donjpierce** [comment]
I can confirm I'm experiencing the same thing with v3.57.0. It started happening right after the CLI update.

**Comment by Piccirello** [comment]
Apologies for this issue. We've released v3.57.1 containing a fix. We're also investigating add automated tests for this functionality to ensure we don't break it again in the future.
